app/web_framework/cluster_base_handler.py

```diff
@@ -192,3 +192,14 @@ def _is_request_origin_allowed(self, request_origin):
         self._logger.debug('Origin "{}" did not match cors_allowed_origins_regex conf value of "{}". '
                            'Not setting Access-Control-Allow-Origin header.', request_origin, allowed_origins_regex)
         return False
+
+
+class ItemNotFoundHandler(ClusterBaseAPIHandler):
+    """
+    ClusterBaseAPIHandler that raises an ItemNotFoundError when it's instantiated.
+    """
+    def initialize(self, route_node=None, **kwargs):
+        super().initialize(route_node)
```

Comment:
Yeah so it's because without defining `initialize` here, we get passed in `cluster_master` or `cluster_slave` as a keyword argument since this handler is being instantiated by the master/slave service. 

Then we call `super().initialize(route_node)` because all of our handlers are expected to be assigned to a route node (without it we get this error):

```
AttributeError: 'ItemNotFoundHandler' object has no attribute '_route_node'
```